recipes/default.rb

```diff
@@ -47,7 +47,7 @@
     )
     notifies :restart, 'service[networking]', :immediately
 
-    if node['ucarp']['interface'].match(/bond/)
+    if node['ucarp']['interface'] ~= /bond/
```

Comment:
I'm having trouble finding any documentation of a `~=` operator. Based on other operators, I'd expect this to be an assignement operation, which, inside an `if` condition, would be a style bug.

It seems likely you meant to use `=~`, which is the pattern match operator. Is this correct?